CspSubmissionPlugin.inc.php

```diff
@@ -1246,10 +1246,10 @@ function TemplateManager_fetch($hookName, $args) {
 
 			return true;
 		} elseif ($args[1] == 'controllers/grid/gridCell.tpl'){
-			if($request->_requestPath == "/ojs/index.php/csp/$$\$call\$$$/grid/files/submission/editor-submission-details-files-grid/fetch-grid" 
-			OR $request->_requestPath == "/ojs/index.php/csp/$$\$call\$$$/grid/files/final/final-draft-files-grid/fetch-grid" 
-			OR $request->_requestPath == "/ojs/index.php/csp/$$\$call\$$$/grid/files/review/editor-review-files-grid/fetch-grid"
-			OR $request->_requestPath == "/ojs/index.php/csp/$$\$call$\$$/grid/files/production-ready/production-ready-files-grid/fetch-grid") { //Busca comentários somente quando grid for de arquivos
+			if(strpos($request->_requestPath, 'files/submission/editor-submission-details-files-grid/fetch-grid')
```

Comment:
Se a string em `$request->_requestPath` começar com files vai retornar zero, isto é problema para esta lógica? Se for deveria colocar comparação de tipo `!== false`

Reply:
Ou posso usar editor-submission-details-files-grid/fetch-grid. Seria melhor né?